sdk/core/azure-core-http-jdk/src/main/java/module-info.java

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import com.azure.core.http.jdk.httpclient.implementation.JdkHttpClientProvider;
+import com.azure.core.http.jdk.implementation.JdkHttpClientProvider;
```

Comment:
Why do we have an `import` in the `module-info`?

Reply:
I guess instead of using the fully qualified class name, the import was used here:

```java
provides com.azure.core.http.HttpClientProvider
        with JdkHttpClientProvider;
```